west.yml

```diff
@@ -118,6 +118,9 @@ manifest:
     - name: edtt
       path: tools/edtt
       revision: dd4dd502ef2fbeced6ef7faaba562a7ddca45632
+    - name: trusted-firmware-m
+      path: modules/tfm
```

Comment:
Not sure if this should instead belong in `modules/tee/tfm`?
@galak @ioannisg @microbuilder

Reply:
Agreed, at least ARC might have their own TEE module, after all

Reply:
@galak @microbuilder does `modules/tee/tfm` make sense to you?

Reply:
`tee` seems fine to me, yes.

Reply:
I moved it to that location.

Reply:
Fixed